### PR TITLE
A3E: KAT MEDICAL RW Stretcher Evac integration into KAT

### PR DESCRIPTION
**Pull request by Battlekeeper**
A3E: KAT MEDICAL RW Stretcher Evac integration into KAT

original mod: https://steamcommunity.com/sharedfiles/filedetails/?id=2445826605

### CODE REVIEW

**Comment by Tomcat-SG** [comment]
Minimal To Do before merging:

* Paths of data which are in the evac mod needs to be changed. They wont work like this if we build them like Kat.
* Using CBA Events instead of Arma PostInit etc.
* Function naming needs a change
* Code optimisation

File: addons/evac/functions/fn_stretcher_attach.sqf [hunk 1]
```diff
@@ -0,0 +1,213 @@
+_h = cursorObject;
```
Comment (Conrimaceogain) [inline addons/evac/functions/fn_stretcher_attach.sqf:1]: Let's get this file renamed to `fnc_stretchter_attach.sqf`

File: addons/evac/functions/fn_stretcher_init.sqf [hunk 2]
```diff
@@ -0,0 +1,4 @@
+["Attachable_Stretcher", ["CONTAINER","CLOTHES"], ["Attach Stretcher"], [], "", [{true},{params ["_unit", "_container", "_item", "_slot", "_params"];_unit == vehicle _unit}], {
```
Comment (Conrimaceogain) [inline addons/evac/functions/fn_stretcher_init.sqf:1]: Same as above: `fnc_stretcher_init.sqf`

File: addons/evac/functions/fn_stretcher_attach.sqf (hunk 1 above)
Comment (Conrimaceogain) [reply addons/evac/functions/fn_stretcher_attach.sqf:1]: And get a comment like this going and clean up the formatting so it's a bit more consistent and then I think should be good to go.
```
/*
 * Author: mharis001
 * Checks if AED Station can be performed on the patient.
 *
 * Arguments:
 * 0: Medic (not used) <OBJECT>
 * 1: Patient <OBJECT>
 *
 * Return Value:
 * Can CPR <BOOL>
 *
 * Example:
 * [player, cursorObject] call kat_circulation_fnc_AEDStationCondition
 *
 * Public: No
 */
```

File: addons/evac/functions/fn_stretcher_init.sqf (hunk 2 above)
Comment (Conrimaceogain) [reply addons/evac/functions/fn_stretcher_init.sqf:1]: And get a comment like this going and clean up the formatting so it's a bit more consistent and then I think should be good to go.
```
/*
 * Author: mharis001
 * Checks if AED Station can be performed on the patient.
 *
 * Arguments:
 * 0: Medic (not used) <OBJECT>
 * 1: Patient <OBJECT>
 *
 * Return Value:
 * Can CPR <BOOL>
 *
 * Example:
 * [player, cursorObject] call kat_circulation_fnc_AEDStationCondition
 *
 * Public: No
 */
```

**Pull request closed without merging**